python/pyspark/mllib/fpm.py

```diff
@@ -68,11 +68,14 @@ def train(cls, data, minSupport=0.3, numPartitions=-1):
         """
         Computes an FP-Growth model that contains frequent itemsets.
 
-        :param data: The input data set, each element contains a
-            transaction.
-        :param minSupport: The minimal support level (default: `0.3`).
-        :param numPartitions: The number of partitions used by
-            parallel FP-growth (default: same as input data).
+         :param data:
```

Comment:
There are extra spaces in the indentations here.  :param should line up with 'Computes' as before